### PR TITLE
libc: complete documentation for stdio functions

### PR DESCRIPTION
**Pull request by damianloew**
Ordering and adding missing docs for stdio functions

DONE: PD-331

### CODE REVIEW

File: libc/functions/f/feof.impl.md [hunk 1]
```diff
@@ -1,24 +1,40 @@
-###Synopsis
+# Synopsis 
+`#include <stdio.h>`</br>
 
-`#include <stdio.h>`
+` int feof(FILE *stream);`</br>
```
Comment (niewim19) [inline libc/functions/f/feof.impl.md:4]:
```suggestion
`int feof(FILE *stream);`</br>
```

File: libc/functions/f/ferror.impl.md [hunk 2]
```diff
@@ -1,27 +1,40 @@
-###Synopsis
+# Synopsis 
+`#include <stdio.h>`</br>
 
-`#include <stdio.h>`
+` int ferror(FILE *stream);`</br>
```
Comment (niewim19) [inline libc/functions/f/ferror.impl.md:4]:
```suggestion
`int ferror(FILE *stream);`</br>
```

File: libc/functions/p/pclose.part-impl.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+# Synopsis 
+`#include <stdio.h>`</br>
+
+` int pclose(FILE *stream);`</br>
```
Comment (niewim19) [inline libc/functions/p/pclose.part-impl.md:4]:
```suggestion
`int pclose(FILE *stream);`</br>
```

File: libc/functions/p/popen.part-impl.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
+# Synopsis 
+`#include <stdio.h>`</br>
+
+` FILE *popen(const char *command, const char *mode);`</br>
```
Comment (niewim19) [inline libc/functions/p/popen.part-impl.md:4]:
```suggestion
`FILE *popen(const char *command, const char *mode);`</br>
```

**Review by niewim19 — CHANGES_REQUESTED** [review]
Common additional space in `Synopsis` parts. Other than that good

**Comment by damianloew** [comment]
> Common additional space in `Synopsis` parts. Other than that good

Thanks, corrected.

**Review by niewim19 — APPROVED** [review]
LGTM

**Pull request merged**